cmake/SetupSphinx.cmake

```diff
@@ -0,0 +1,65 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+
+# While we cannot fully support Sphinx+Breathe yet because of some
+# upstream bugs in Sphinx (Sphinx issues #7367 and #7368) we have
+# support for using Sphinx once the issues are resolved. We have
+# a SpECTRE issue #2138 for tracking Sphinx progress.
```

Comment:
links to the issue would be useful (the number without a repo name is not necessarily unique)